texar/torch/run/executor.py

```diff
@@ -1424,6 +1413,10 @@ def test(self, dataset: OptionalDict[DataBase] = None):
                 self._test_tracker.stop()
 
             self._fire_event(Event.Testing, True)
+
+        # close the log files
+        self._close_files()
```

Comment:
And maybe move this after `self.model.train(...)` as well, just to be consistent.

Reply:
Done in e175fa6